ledger/ledger-on-memory/BUILD.bazel

```diff
@@ -179,20 +179,13 @@ conformance_test(
         "--additional=MultiPartySubmissionIT",
         "--additional=ContractIdIT:RejectV0,ContractIdIT:AcceptSuffixedV1,ContractIdIT:AcceptNonSuffixedV1",
         "--additional=ParticipantPruningIT",
-        "--additional=AppendOnlyCommandDeduplicationParallelIT",
```

Comment:
Thanks for fixing this 👍

Reply:
I am surprised the earth was still spinning with these lines in the build file.